File: JAVA PROGRAMMING/Test7.java
// 7. SWAPPING OF TWO VARIABLES' VALUE USING THIRD VARIABLE
import java.util.Scanner;
public class Test7 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Num1: ");
        int num1 = in.nextInt();

        System.out.println("Enter Num2: ");
        int num2 = in.nextInt();

        System.out.println("Num1: "+num1);
        System.out.println("Num2: "+num2);

        int temp = num1;
        num1 = num2;
        num2 = temp;
        System.out.println("Num1: "+num1);
        System.out.println("Num2: "+num2);

        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test7.ts
// 7. SWAPPING OF TWO VARIABLES' VALUES USING A THIRD VARIABLE
const tempSwap = (ip1: number, ip2: number): void => {
    console.log(`Current Num1 Value: ${ip1}`);
    console.log(`Current Num2 Value: ${ip2}`);
    let temp:number = 0;
    temp = ip1;
    ip1 = ip2;
    ip2 = temp;
    console.log(`Updated Num1 Value: ${ip1}`);
    console.log(`Updated Num2 Value: ${ip2}`);
}
tempSwap(10, 20);